dispatch/BESCatalogList.cc

```diff
@@ -68,32 +68,31 @@ BESCatalogList *BESCatalogList::d_instance = 0;
 BESCatalogList *
 BESCatalogList::TheCatalogList()
 {
-    if (d_instance == 0) initialize_instance();
-
-    return d_instance;
+    static BESCatalogList catalog;
+    return &catalog;
 }
 
-/**
- * private static that only get's called once by using pthread_once and
- * pthread_once_t mutex.
- */
-void BESCatalogList::initialize_instance()
-{
-    d_instance = new BESCatalogList;
-#ifdef HAVE_ATEXIT
-    atexit(delete_instance);
-#endif
+///**
```

Comment:
Just remove this. However, if you think old code sections should be retained for a while, wrap them in 
#if 0
...
#endif

Same for the sections below.

Reply:
kk
on it